scilpy/image/tests/test_volume_operations.py

```diff
@@ -201,6 +202,67 @@ def test_resample_volume():
     assert resampled_img.affine[0, 0] == 3
 
 
+def test_reshape_volume_pad():
+    # 3D img
+    img = nib.Nifti1Image(
+        np.arange(1, (3**3)+1).reshape((3, 3, 3)).astype(float),
```

Comment:
I see ((3**3) * 2)+1 and (3**3)+1 , what is your spacing guideline ?

Reply:
I think he means that this line (`(3**3)+1`) and line 228 (`(3**3) * 2)+1`) are not the same.

Reply:
Ah. I adjusted the comment to make it clearer.

Reply:
Fixed in [30a5515](https://github.com/scilus/scilpy/pull/1019/commits/30a551512eadabb440dde90c9fed4dd0ae8db767)